homeassistant/components/kostal_plenticore/const.py

```diff
@@ -688,11 +690,59 @@
         {ATTR_UNIT_OF_MEASUREMENT: PERCENTAGE, ATTR_ICON: "mdi:battery-negative"},
         "format_round",
     ),
-    (
+]
+
+# Defines all entities for switches.
+#
+# Each entry is defined with a tuple of these values:
+#  - module id (str)
+#  - process data id (str)
+#  - entity name suffix (str)
+#  - on Value (str)
+#  - on Label (str)
+#  - off Value (str)
+#  - off Label (str)
+SWITCH = namedtuple(
```

Comment:
We prefer defining named tuples in the modern way as we do below for `SelectData` including type annotations.

Reply:
Fixed in https://github.com/home-assistant/core/pull/59998